study.md

```diff
@@ -66,15 +66,15 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+<!-- No it should not becuase it is all seperate fillins for the star object. So changing sol should only change sol. -->
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
+<!-- Yes it will effect the nearest_star, becuase nearest_star is comparing all of the objects declared under the star object. -->
```

Comment:
`nearest_star` and `the_sun` will affect each other because both are references to the same object.